platform/pwl.h

```diff
@@ -125,7 +125,7 @@ struct wl_buffer*
             pwl_display_egl_create_buffer_from_image (const PwlDisplay*, EGLImage);
 bool        pwl_display_egl_has_broken_buffer_from_image (const PwlDisplay*);
 
-PwlXKBData* pwl_display_xkb_get_data (PwlDisplay*);
+PwlXKBData* pwl_display_xkb_get_data (PwlSeat*);
```

Comment:
This operates on a `PwlSeat`, therefore it should be renamed to `pwl_seat_xkb_get_data()`.

Reply:
done.